modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -298,54 +298,63 @@ function add_dependency_permutations(o, deps) {
 		o.depends(res[i]);
 }
 
+// Define a class CBIWifiFrequencyValue that extends form.Value
 var CBIWifiFrequencyValue = form.Value.extend({
+	// Declare an RPC method to get the frequency list for a given device
 	callFrequencyList: rpc.declare({
 		object: 'iwinfo',
 		method: 'freqlist',
-		params: [ 'device' ],
+		params: ['device'],
 		expect: { results: [] }
 	}),
 
+	// Load method to fetch WiFi device details and frequency list
 	load: function(section_id) {
 		return Promise.all([
-			network.getWifiDevice(section_id),
-			this.callFrequencyList(section_id)
+			network.getWifiDevice(section_id), // Get WiFi device info
+			this.callFrequencyList(section_id) // Call frequency list
 		]).then(L.bind(function(data) {
-			this.channels = {
-				'2g': L.hasSystemFeature('hostapd', 'acs') ? [ 'auto', 'auto', true ] : [],
-				'5g': L.hasSystemFeature('hostapd', 'acs') ? [ 'auto', 'auto', true ] : [],
-				'6g': L.hasSystemFeature('hostapd', 'acs') ? [ 'auto', 'auto', true ] : [],
+		// Initialize channels for different bands
+		this.channels = {
+				'2g': L.hasSystemFeature('hostapd', 'acs') ? ['auto', 'auto', true] : [],
+				'5g': L.hasSystemFeature('hostapd', 'acs') ? ['auto', 'auto', true] : [],
+				'6g': L.hasSystemFeature('hostapd', 'acs') ? ['auto', 'auto', true] : [],
 				'60g': []
 			};
 
-			for (var i = 0; i < data[1].length; i++) {
-				if (!data[1][i].band)
-					continue;
+				// Populate channels based on the frequency data received
```

Comment:
Any reason this indent is deeper?

Reply:
dunno .. I changed one level up ... please note that I haven't added any new code I just added to the existing code the 11be capability.  I also added @danpawlik